package.json

```diff
@@ -58,6 +58,8 @@
     "pretest": "npm run lint",
     "props-table": "lerna run props-table",
     "postinstall": "link-parent-bin && npm run compile && npm run bootstrap",
```

Comment:
shouldn't you update this line to be the following:

"postinstall": "node scripts/post-install/index.js",

Reply:
Good call, [updated](https://github.com/cerner/terra-framework/pull/21/commits/9f51c3b1127464c0f6a3b1a52638ff714f011866)